client/.env

```diff
@@ -0,0 +1,2 @@
+REACT_APP_GRAPHQL_BASE_URL=http://localhost:4000/proxy/graphql
```

Comment:
These ones could go to `.env.development` (and that move can of course wait until we have a "production" environment)

Reply:
I think that I'll wait to the point when we are talking about deployment